app/Community/Components/UserProfileMeta.php

```diff
@@ -28,58 +29,255 @@ public function __construct(
+        array $userJoinedGamesAndAwards
+    ): array {
+        $hardcorePoints = $userMassData['TotalPoints'] ?? 0;
+        $softcorePoints = $userMassData['TotalSoftcorePoints'] ?? 0;
+        $preferredMode = $softcorePoints > $hardcorePoints ? 'softcore' : 'hardcore';
+
+        $recentPointsEarned = $this->calculateRecentPointsEarned($user, $preferredMode);
+
+        // Total games beaten
+        $totalGamesBeaten = (int) PlayerStat::where('user_id', $user->ID) // keep in mind, this is hardcore only.
+            ->where('system_id', null)
+            ->sum('value');
+        $totalGamesBeatenStat = [
+            'label' => 'Total games beaten',
+            'value' => localized_number($totalGamesBeaten),
+            'isMuted' => !$totalGamesBeaten,
+            // TODO: href to 'ranking.beaten-games' for a specific user.
+        ];
+
+        // Started games beaten
+        $startedGamesBeatenPercentage = $this->calculateAverageFinishedGames($this->userJoinedGamesAndAwards);
+        $startedGamesBeatenPercentageStat = [
+            'label' => 'Started games beaten',
+            'value' => $startedGamesBeatenPercentage . '%',
+            'isMuted' => $hardcorePoints === 0 && $softcorePoints === 0,
+        ];
+
+        // Points earned in the last 7 days
+        $pointsLast7DaysStat = [
+            'label' => 'Points earned in the last 7 days',
+            'value' => localized_number($recentPointsEarned['pointsLast7Days']),
+            'isMuted' => !$recentPointsEarned['pointsLast7Days'],
+        ];
+
+        // Points earned in the last 30 days
+        $pointsLast30DaysStat = [
+            'label' => 'Points earned in the last 30 days',
+            'value' => localized_number($recentPointsEarned['pointsLast30Days']),
+            'isMuted' => !$recentPointsEarned['pointsLast30Days'],
+        ];
+
+        // Average points per week
+        $averagePointsPerWeek = $this->calculateAveragePointsPerWeek(
+            $this->userMassData['MemberSince'],
+            // Calculate based on the user's primary playing mode.
+            $preferredMode === 'softcore' ? $softcorePoints : $hardcorePoints,
+        );
+        $averagePointsPerWeekStat = [
+            'label' => 'Average points per week',
+            'value' => localized_number($averagePointsPerWeek),
+            'isMuted' => !$averagePointsPerWeek,
+        ];
+
+        // Average completion percentage
+        $averageCompletionStat = [
+            'label' => 'Average completion',
+            'value' => $this->averageCompletionPercentage . "%",
+            'isMuted' => $this->averageCompletionPercentage === '0.0',
+        ];
+
+        return array_merge(
+            compact(
+                'averageCompletionStat',
+                'averagePointsPerWeekStat',
+                'pointsLast30DaysStat',
+                'pointsLast7DaysStat',
+                'startedGamesBeatenPercentageStat',
+                'totalGamesBeatenStat',
+            ),
```

Comment:
Suggest reordering these so the beaten/completion stuff is in one column, and the points stuff is in the other.

Current:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/8f191af8-ea26-4645-b738-4b6b354ac53c)

Proposed:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/735ea056-8000-4d31-9f4e-2d47027fb881)

This also flows better in mobile.
Current:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/792c5929-401b-45c7-b2af-d11278f3d500)

Proposed:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/6fb554e6-0090-4828-b6e1-1a2b038e807d)

Reply:
No problem, I've changed the sort order for both the mixed and non-mixed "see more" content in latest.